test/controllers/reviews_controller_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class ReviewsControllerTest < ActionController::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+
+  # test "should get new" do
```

Comment:
Looking at the route for this may help you see what information you will need to put into params to test this method. Given that it is a new_product_review, I would imagine that at the very least you will need to pass it a product.